partcad/src/partcad/project_factory_git.py

```
@@ -86,82 +115,109 @@ def _clone_or_update_repo(self, repo_url, cache_dir=None):
+                                patience,
+                            )
+                            time.sleep(patience)
+                        else:
+                            pc_logging.error("Failed to update repo after %s retries" % attempt)
+                            # Fall back to using the previous copy
+                else:
+                    # Clone the repository if it's not cached yet.
+                    try:
+                        pc_logging.info("Cloning the GIT repo: %s" % self.import_config_url)
+                        repo = Repo.clone_from(repo_url, cache_path)
+                        if not self.import_revision is None:
```

Ruff (0.8.2)

Test for object identity should be `is not`

Convert to `is not`

(E714)